### PR TITLE
predownload HITRAN database

### PR DESCRIPTION
**Pull request by erwanp**
First attempt !

- modified the Docker file
- update Radis to 0.11.1

### CODE REVIEW

**Comment by suzil** [comment]
Let's try it!

**Comment by erwanp** [comment]
From the test session yesterday, I'm quite sure CO2 (and therefore all HITRAN molecules) are not cached as I wanted them to (computing a CO2 spectrum with default conditions should be extremelly fast, here Radis-App gets stucks which suggests it's downloading & parsing the full database). 

[x] However, CO works, I do not explain this. Is it because you happen to test it after the launch, and therefore it ends up being cached ? 

[x] tried NO (small molecule), it works too.
[x] tried H2O (large molecule), it does not

**Comment by erwanp** [comment]
It's still possible to switch back to the old mode, where database were not pre-downloaded, by using the "range" parameter : 

```python
calc_spectrum(..., databank=("hitran", "full"))   # predownload full database & isotopes (once for all)
calc_spectrum(..., databank=("hitran", "range"))   # download only the required the range & isotope
```

But it would be much more efficient if the database was cached.

File: lib/radis-lambda/download_hitran.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+Download some molecules from HITRAN data, parse & cache them, to make 
+sure the example notebooks run fast.
+"""
+
+from radis.io.hitran import fetch_hitran
+
+for molecule in [
```
Comment (erwanp) [inline lib/radis-lambda/download_hitran.py:9]: @suzil do we have an equivlaent of this pre-download script on the new machine?

Comment (suzil) [reply lib/radis-lambda/download_hitran.py:9]: I ran the script on the server now so it should be working as expected from now.

Comment (suzil) [reply lib/radis-lambda/download_hitran.py:9]: Code for the server added in this PR: https://github.com/suzil/radis-app/pull/568

Comment (erwanp) [reply lib/radis-lambda/download_hitran.py:9]: Nice ! I'm adding a `download_geisa.py` script there; but I do not know where to call it (maybe you have to run it on the server bash ? )

Comment (suzil) [reply lib/radis-lambda/download_hitran.py:9]: Yes, I'll run it manually.

**Pull request merged**